xbmc/platform/linux/peripherals/PeripheralBusUSBLibUdev.cpp

```diff
@@ -108,7 +108,12 @@ bool CPeripheralBusUSB::PerformDeviceScan(PeripheralScanResults &results)
 
     if (bContinue)
     {
-      dev = udev_device_get_parent(udev_device_get_parent(parent));
+      // do not call udev_device_get_parent(udev_device_get_parent(dev))
```

Comment:
Comments not needed as it's included in the commit message

Reply:
Dropped to comments.